modules/ROOT/pages/_partials/dataweave-formats-flatfile-config.adoc

```diff
@@ -14,8 +14,7 @@ DataWeave accepts properties that provide instructions for reading input data in
 | Parameter | Type | Default | Description
 | `enforceRequires` | `Boolean` | `false` | Error if required value missing.
   Valid values are `true` or `false`.
-| `missingValues` | `String` | `nulls` for copybook schema, `spaces` otherwise | Fill character used to represent missing
-values. To represent missing values in the input data, you can use:
+| `missingValues` | `Boolean` | `nulls` for copybook schema, `spaces` otherwise | Fill character used to represent missing values. To activate a non-default setting, set the `useMissCharAsDefaultForFill` property to `true`, and use one of the following values to `missingValues`:
```

Comment:
```suggestion
| `missingValues` | `String` | `nulls` for copybook schema, `spaces` otherwise | Fill character used to represent missing values. To activate a non-default setting, set the `useMissCharAsDefaultForFill` property to `true`, and use one of the following values to `missingValues`:
```